lymph/unilateral.py

```diff
@@ -80,12 +80,16 @@ def __init__(self, graph: Dict[Tuple[str], Set[str]], **_kwargs):
         if len(name_list) != len(name_set):
             raise ValueError("No tumor and LNL can have the same name")
 
-        self.nodes = []        # list of all nodes in the graph
-        self.tumors = []       # list of nodes with type tumour
-        self.lnls = []         # list of all lymph node levels
+        self.nodes = []                     # list of all nodes in the graph
+        self.tumors = []                    # list of nodes with type tumour
+        self.lnls = []                      # list of all lymph node levels
+        self.states = states
+        if states == 3:
+            self.microscopic_parameter = 0     # holds the micrscropic spread scaling parameter
+            self.growth_parameter = 0          # holds the growth probability parameter for a LNL to change from micrscopic to macroscopic involvement

```

Comment:
I would not hard-code these parameters here. It is enough to write a nice `assign_parameters` method or sth like that, which handles assigning the correct parameters to the `Edge` instances. See also #46 for more thoughts on that.

Reply:
I already got rid of this. Right now it is an assign parameters to edge and node method